src/app/twiglet-graph/handleUserStateChanges.spec.ts

```diff
@@ -134,9 +135,10 @@ describe('TwigletGraphComponent:handleUserStateChanges', () => {
     });
   });
 
-  describe('textToFilterOn', () => {
-    it('greys out nodes that do not match search parameters', () => {
+  describe('textToFilterOn and filterEntities', () => {
+    it('greys out nodes that do not match search and filterEntities parameters', () => {
       response.textToFilterOn = 'first';
+      response.filterEntities = ['ent1'];
```

Comment:
Not sure if this belongs here tbh. If it didn't change the results of the tests (like, if this line was removed the test would still pass) then it isn't truly testin the functionality. Might be worth copying and pasting the test and checking to see if they grey out if only the filterEntities is active.